tests/framework/OutStreams/test_legend.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" ?>
```

Comment:
This input file does not work in my machine. I do not get any legend. 
![legendtest](https://cloud.githubusercontent.com/assets/9202671/25288969/005d5252-2685-11e7-9cb8-3dc943447ec2.png)


>>> import matplotlib
>>> matplotlib.__version__
'1.5.1'